src/views/HighscoreView.vue

```diff
@@ -0,0 +1,222 @@
+          <th class="score-table_th">right</th>
+          <th class="score-table_th">score</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td></td>
+          <td colspan="4"><hr /></td>
+        </tr>
+        <tr
+          v-for="(gamer, index) of toggleHighscoreDetails"
+          :key="gamer.id"
+          :class="{ 'score-table_td-current': gamer.id === store.highscoreID }"
+        >
+          <td class="score-table_td">{{ index + 1 }}.</td>
+          <td class="score_gamer-name score-table_td">{{ gamer.name }}</td>
+          <td class="score-table_td">{{ gamer.elapsedTime }}</td>
+          <td class="score-table_td">
+            {{ gamer.result[0] }} / {{ gamer.result[1] }}
+          </td>
+          <td class="score-table_td">{{ gamer.highscore }}</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+import { useQuizStore } from "@/stores/quizStore.js";
+
+export default {
+  name: "HighscoreView",
+  setup() {
+    const store = useQuizStore();
+    return { store };
+  },
+  data() {
+    return {
+      currentHighscoreType: 5,
+      highscoreData: [],
+      sortedHighscoreData: [],
+      currentEntry: "",
+    };
+  },
+  computed: {
+    toggleHighscoreDetails() {
+      return this.sortedHighscoreData.filter((gamer) => {
+        return gamer.result[1] === this.currentHighscoreType;
+      });
+    },
+  },
+  methods: {
+    toggleHighscoreType(n) {
+      this.currentHighscoreType = n;
+    },
+    getHighscoreScore() {
+      this.highscoreData.forEach((gamer) => {
+        const timePerQuestion = gamer.elapsedTime / gamer.result[1];
+        let factor = 1;
+        //if average time per question < 15s -> factor = 10
+        //for every 5 seconds more, factor reduces at 1
+        //minimum factor = 1
+        for (let i = 0; i <= 9; i++) {
+          if (timePerQuestion <= 15 + i * 5) {
+            factor = 10 - i;
+            i = 9;
+          }
+        }
+        let bonus = 0;
+        if (gamer.passedRatio === "100%") {
+          bonus = 300;
+        }
+        gamer.highscore = gamer.result[0] * 10 * factor + bonus;
+      });
+    },
+    sortByHighscore() {
+      //sort highscore values
+      const sortScore = this.highscoreData
+        .map((gamer) => {
+          return gamer.highscore;
+        })
+        .sort((a, b) => b - a);
+      //push gamer data in the right order
+      sortScore.forEach((score) => {
+        this.sortedHighscoreData.push(
+          this.highscoreData.filter((gamer) => {
+            return gamer.highscore === score;
+          })[0]
+        );
+      });
+    },
+    getCurrentEntry() {
+      this.currentEntry = this.highscoreData.filter((game) => {
+        return game.id === this.store.highscoreID;
+      })[0];
+      this.currentHighscoreType = this.currentEntry.total;
+    },
+  },
+  async created() {
+    const response = await fetch(
+      "https://22-november.api.cbe.uber.space/highscore"
```

Comment:
ich habe in meiner Branch die Basis URL bis `...space/` in den Store gespeichert. Das können wir hier noch anpassen, nachdem gemerged wurde.